WTWSMS/decisions/minor_decisions.txt

```diff
@@ -630,7 +637,7 @@ decisions = {
 				factor = 0.001             # Only a small minority of Christians ever go on a pilgrimage
 			}
 			modifier = {
-				factor = 3
+				factor = 50
```

Comment:
They are more frequent than in vanilla with your change, which they shouldn't be.